app/main.py

```diff
@@ -132,6 +132,20 @@ async def delete(request):
     status = await (dqueue.cancel(ids) if where == 'queue' else dqueue.clear(ids))
     return web.Response(text=serializer.encode(status))
 
+@routes.get(config.URL_PREFIX + 'history')
+async def list_queue(_):

```

Comment:
Let's rename `_` to `request` just for consistency with the other calls

Reply:
Alright i'll make a patch